pages/index.js

```diff
@@ -1,3 +1,172 @@
+  },
+  {
+    id: "u-030234Lbc",
+    name: "NHS",
+    email: "NHS@example.com",
+    image: "www-image.url.de",
+  },
+  {
+    id: "u-318Lca",
+    name: "quer",
+    email: "quer@example.com",
+    image: "www-image.url.de",
+  },
+  {
+    id: "u-318Lci",
+    name: "wear",
+    email: "wear@example.com",
+    image: "www-image.url.de",
+  },
+  {
+    id: "u-040Lbc",
+    name: "SRF",
+    email: "SRF@example.com",
+    image: "www-image.url.de",
+  },
+  {
+    id: "u-318Lcid",
+    name: "wear",
+    email: "wear@example.com",
+    image: "www-image.url.de",
+  },
+  {
+    id: "u-318Lciad",
+    name: "Asie",
+    email: "Asie@example.com",
+    image: "www-image.url.de",
+  },
+];
+
+const graffitiPosts = [
+  {
+    id: "g-14",
+    userID: "u-213Lbc",
+    location: "Copenhagen",
+    image: "/images/bates-truck.png",
+    comment:
+      "Yoo check out this Truck i busted haha Senior and still bombing!!!",
+  },
+  {
+    id: "g-1234",
+    userID: "u-030Lbc",
+    location: "Berlin",
+    image: "/images/gms-train.png",
+    comment: "Yoo check out my newest work!!!",
+  },
+  {
+    id: "g-12",
+    userID: "u-213Lbc",
+    location: "Hamburg",
+    image: "/images/bates-wall.png",
+    comment: "Yoo looks fire!!!",
+  },
+  {
+    id: "g-040Lbca",
+    userID: "u-040Lbca",
+    location: "Hamburg",
+    image: "/images/srf-crew.png",
+    comment: "SRF for Life",
+  },
+  {
+    id: "g-040Lbc",
+    userID: "u-040Lbc",
+    location: "Hamburg",
+    image: "/images/emra.png",
+    comment: "SRF for Life",
+  },
+  {
+    id: "g-318Lca",
+    userID: "u-318Lca",
+    location: "Itzehoe",
+    image: "/images/quer.png",
+    comment: "#Long live Alsen",
+  },
+  {
+    id: "g-318Lci",
+    userID: "u-318Lci",
+    location: "Itzehoe",
+    image: "/images/wear-alsen.png",
+    comment: "#Long live Alsen",
+  },
+  {
+    id: "g-318Lcid",
+    userID: "u-318Lci",
+    location: "Itzehoe",
+    image: "/images/wear-image.png",
+    comment: "#Long live Alsen",
+  },
+  {
+    id: "g-318Lcidad",
+    userID: "u-318Lciad",
+    location: "Itzehoe",
+    image: "/images/asie-wagon.png",
+    comment: "#Long live Alsen",
+  },
+  {
+    id: "g-318Lcidaded",
+    userID: "u-318Lciad",
+    location: "Itzehoe",
+    image: "/images/asie-wall.png",
+    comment: "#Long live Alsen",
+  },
+];
+
+export default function DiscoverPage() {
+  return (
+    <>
+      <h1>Graffstar</h1>
+      {graffitiPosts.map((graffiti) => {
+        const currentUser = users.find((user) => user.id === graffiti.userID);
+        return (
+          <section key={graffiti.id}>
+            <h2>Post by {currentUser.name}</h2>
+            <p>Location: {graffiti.location}</p>
+            <Image
+              src={graffiti.image}
+              alt={`Graffiti from ${currentUser.name}`}
+              width={250}
+              height={250}
+            />
+            <p>{graffiti.comment}</p>
+          </section>
+        );
+      })}
+      <section>
```

Comment:
It's not necessary to wrap your `navbar` into a `section`, this could lead to styling problems later on.